packages/connector-jsdom/src/connector.ts

```diff
@@ -45,6 +45,8 @@ import { Engine } from 'hint/dist/src/lib/engine';
 import isHTMLDocument from 'hint/dist/src/lib/utils/network/is-html-document';
 import createHTMLDocument from 'hint/dist/src/lib/utils/dom/create-html-document';
 import traverse from 'hint/dist/src/lib/utils/dom/traverse';
+
+import { HTMLEvents } from '@hint/parser-html';
```

Comment:
Because we are using only the type maybe use [`import types`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-9.html#import-types) in loine 89?

Reply:
Yep, just using the type definitions, but of course an original document will only be set up if HTML parsing is enabled (as we'd only trigger the relevant events in that case).

I'll make the update to inline the type import.

Reply:
What are the benefits of using `import types`?

Reply:
I think we should start using those when we only need something for types to make it clear that it's a dev dependency and not required at run time. I was confused with @antross comment:

> All connectors now use `parser-html` to obtain an original source document

when I was reading the code and just seeing the types being used. So basically for clarity 😊

Reply:
I'm asking, because for me it was weird to see that, I think it makes the code less readable.

Reply:
But the sections import is cleaner because you only have the imports you need to run the code and not something that is going to be use one time. If the types are going to be used more than once I think we should stick to what we are doing. If it is just once then we should evaluate using this more.